src/components/Molecules/ShareButton/ShareButton.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../../Atoms/SocialIcons/Icon/Icon';
+import PopUpHelper from '../../../utils/ShareButton/SharePopUpHelper';
+import ShareUrlHelper from '../../../utils/ShareButton/ShareUrlHelper';
+import ShareIcons from './assets/ShareIcons';
+
+import {
+  Wrapper,
+  StyledParagraph,
+  StyledList,
+  StyledItem
+} from './ShareButton.style';
+
+const handleShare = (e, typeOfShare, urlToShare) => {
+  e.preventDefault();
+
+  // Pass the current page's URL and the type of share to our helper function
+  const shareUrl = ShareUrlHelper(urlToShare, typeOfShare);
+
+  // Use our helper function for pop-up position issues on dual-screen setups
+  PopUpHelper(shareUrl, 550, 420);
+};
+
+/* Share Button component to handle FB and Twitter sharing */
+const ShareButton = ({ campaign, copy, urlToShare, ...restProps }) => {
+  return (
+    <Wrapper>
+      <StyledParagraph>{copy}</StyledParagraph>
```

Comment:
We tend not to use prefixes like "Styled" + Component name.

Reply:
Rightio, was just looking at another component for guidance but can rename it to just.. Paragraph? I figured explicitly naming it to reflect what it is is a good approach?

Reply:
In the context it is just a Paragraph component.. It would be a lot of Styled if we add it to all styled components.. just a convention .. we can think about our own standard for styled components naming.

Reply:
Yeah, that's true; if we do it to _everything_, it loses all meaning.